.gitlab.mk

```diff
@@ -117,7 +117,7 @@ vms_shutdown:
 GIT_DESCRIBE=$(shell git describe HEAD)
 MAJOR_VERSION=$(word 1,$(subst ., ,$(GIT_DESCRIBE)))
 MINOR_VERSION=$(word 2,$(subst ., ,$(GIT_DESCRIBE)))
-BUCKET="$(MAJOR_VERSION).$(MINOR_VERSION)"
+BUCKET="series-$(MAJOR_VERSION)"
```

Comment:
I propose the flat layout for tarballs, without splitting for the series: `https://download.tarantool.org/tarantool/src/*.tar.gz`. I would even copy older tarballs there for convenience of ebuild / pkgbuild / port makefiles writters.

I'll add this point into the release policy RFC.

Reply:
Fixed